cadence/transactions/adminTransferProceeds.cdc

```diff
@@ -0,0 +1,18 @@
+import NFTPawnshop from "../contracts/NFTPawnshop.cdc"
+import NonFungibleToken from "../contracts/NonFungibleToken.cdc"
+import Domains from "../contracts/Domains.cdc"
+
+transaction(recipient: Address) {
```

Comment:
Nice. Admin should be able to send the proceeds to the address of his choice.

Reply:
Yes, exactly. It can be achieved with a transaction like this:
```bash
flow transactions send ./cadence/transactions/adminTransferProceeds.cdc 0x179b6b1cb6755e31 --network=emulator --signer=emulator-account
```